builtin/logical/database/rollback.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"errors"
 
+	"github.com/hashicorp/vault/sdk/database/dbplugin"
+
```

Comment:
There's an extra newline here.

Reply:
<img src="https://media.giphy.com/media/xT5LMpBWU96Yivdfeo/giphy.gif" width=250>
Whoops - looks like it went through in the original PR =/